test/unit_Polyhedron.jl

```diff
@@ -112,27 +112,26 @@ if test_suite_polyhedra
         @test σ(d, p_unbounded) == N[Inf, 0]
         @test ρ(d, p_unbounded) == N(Inf)
         p_infeasible = HPolyhedron([LinearConstraint(N[1], N(0)),
-                                  LinearConstraint(N[-1], N(-1))])
+                                    LinearConstraint(N[-1], N(-1))])
+        @test isempty(p_infeasible)
         @test_throws ErrorException σ(N[1], p_infeasible)
 
         # intersection
-        # TODO these polyhedra are empty. do the tests make any sense?
-        A = [N(0) N(1); N(1) N(0); N(2) N(2)]
-        b = N[0, 0, 1]
+        A = [N(0) N(1); N(1) N(0)]
+        b = N[1, 1]
         p1 = HPolyhedron(A, b)
-        A = [N(0) N(-1); N(-1) N(0); N(1) N(1)]
-        b = N[-0.25, -0.25, 0]
+        A = [N(0) N(-1); N(-1) N(0)]
+        b = N[0, 0]
         p2 = HPolyhedron(A, b)
-        cap = intersection(p1, p2)
-        # currently broken, see #565
+        intersection(p1, p2)
 
         # intersection with polytope
         A = [N(1) N(0); N(0) N(1); N(-1) N(-1)]
         b = N[1, 1, -1]
         p = HPolyhedron(A, b)
         b = BallInf(N[1, 1], N(0.5))
-        cap = intersection(p, b)
-        cap = intersection(b, p)
+        intersection(p, b)
+        intersection(b, p)
```

Comment:
the intention was to test something here or just to check that `intersection` can dispatch on `HPolyhedron` and `BallInf`?

Reply:
I do not know. I just removed the return value because we never used it :wink:

Reply:
😆  we could check that the obtained sets are the same, for instance

Reply:
Okay, see 1531.